src/libs/API.js

```diff
@@ -296,6 +295,33 @@ function reauthenticate(command = '') {
         });
 }
 
+/**
+ * Calls the command=Authenticate API with an accountID, validateCode, and optional 2FA code. This is used specifically
+ * for sharing sessions between e.com and this app. It will return an authToken that is used for initiating a session
+ * in this app. This API call doesn't have any special handling (like retries or special error handling).
+ *
+ * @param {Object} parameters
+ * @param {String} parameters.accountID
+ * @param {String} parameters.validateCode
+ * @param {String} [parameters.twoFactorAuthCode]
+ * @returns {Promise<unknown>}
```

Comment:
NAB / just curious but why `Promise<unknown>`?

Reply:
I think PHPStorm did that automatically, so I'm not sure why! I'll remove it